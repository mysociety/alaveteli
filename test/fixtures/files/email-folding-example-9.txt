Dear Mr Cross

Freedom of Information Request Reference No: 2008040590

Yours sincerely

MICHAEL HEGARTY

FOI Officer

**********************************************************************************************
Please Note: Incoming and Outgoing E-mail messages are routinely monitored
for compliance with our policy on the use of electronic communications.

Interested in Occupational Health & Safety information?
Please visit the HSE website at the following address to keep yourself up
to date

www.hse.gov.uk

Or contact HSE Infoline on 0845 345 0055 or email [HSE request email]

**********************************************************************************************
The original of this email was scanned for viruses by the Government
Secure Intranet virus scanning service supplied by Cable&Wireless in
partnership with MessageLabs. (CCTM Certificate Number 2007/11/0032.) On
leaving the GSi this email was certified virus free.
Communications via the GSi may be automatically logged, monitored and/or
recorded for legal purposes.
