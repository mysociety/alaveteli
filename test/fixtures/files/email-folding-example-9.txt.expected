Dear Mr Cross

Freedom of Information Request Reference No: 2008040590

Yours sincerely

MICHAEL HEGARTY

FOI Officer
